bb-aem-common/src/main/java/com/cognifide/qa/bb/aem/AemCommonModule.java

```diff
@@ -53,4 +58,20 @@ public void init(Properties properties) {
     });
   }
 
+  /**
+   * Provider method that produces a CloseableHttpClient instances configured with URL, login and password
+   * taken from property files.
+   *
+   * @param url      IP address of Author instance, automatically set to author.ip property.
+   * @param login    Author login, automatically set to author.login property.
+   * @param password Author password, automatically set to author.password property.
+   * @return New instance of CloseableHttpClient.
+   */
+  @Provides
+  public CloseableHttpClient getAuthorHttpClient(@Named(AemConfigKeys.AUTHOR_IP) String url,
+      @Named(AemConfigKeys.AUTHOR_LOGIN) String login,
+      @Named(AemConfigKeys.AUTHOR_PASSWORD) String password) {
+    return new HttpClientProvider(login, password, url).get();
```

Comment:
This binds a specific configuration: an HTTP client for  AEM author thus moving it to `bb-core` is a no-go. As a user if you're using only `bb-core` you can define your own bindings in the project.

As for making `bb-reports` independent of any modules related to AEM: :+1: